docs/cli.md

```diff
@@ -227,6 +227,9 @@ option is used.
 * `--dev-only`: Only install dev dependencies. (**Deprecated**)
```

Comment:
AFAIR the `--dev-only` option was never contained within a release. So I think we can remove it now already or at least with final 1.2 release.

Reply:
I think you are correct, however I'd prefer to do the removal in a separate PR, ideally after the next pre-release prior to 1.2 along with default.